Desktop/AddTransactionWindow.cs

```diff
@@ -76,5 +76,20 @@ private void addNewTransactionButton_Click(object sender, EventArgs e)
             }
             this.Close();
         }
+
+        private void transactionDate_ValueChanged(object sender, EventArgs e)
+        {
+
+        }
+
+        private void transactionAmount_TextChanged(object sender, EventArgs e)
+        {
+
+        }
+
+        private void transactionDetailsReasons_TextChanged(object sender, EventArgs e)
+        {
+
+        }
```

Comment:
What are these methods made for?

Reply:
These are automatically made

Reply:
These also can be deleted as the ones written above.

Reply:
I will delete them